eiffel-syntax-and-usage/the-links-object.md

```diff
@@ -0,0 +1,40 @@
+# The Links Object
+The __links__ object is an array of trace links to other Eiffel events. These trace links by definition always reference backwards in time – it is only possible to reference an event that has already occured. Each trace link is a tuple consisting of a type and a UUID corresponding to the __meta.id__ of the target event, on String format.
```

Comment:
I would rather skip the copy-pasted description of the links object and simply reference the eiffel repository, e.g. "This document describes the additions to the __links__ object in the Operations extension to the Eiffel protocol. For a full description, examples and explanation of the __links__ object, please see..."